From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: George T Kramer <george.t.kramer@intel.com>
Date: Mon, 16 Mar 2020 13:08:36 -0700
Subject: [PATCH] Remove check for existence of config file

Sane defaults are already provided by koji.  Use these for stateless
configuration.
---
 koji/__init__.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/koji/__init__.py b/koji/__init__.py
index c05cc1b..370fabe 100644
--- a/koji/__init__.py
+++ b/koji/__init__.py
@@ -2007,9 +2007,7 @@ def read_config(profile_name, user_config=None):
     config = read_config_files(configs)
 
     # Load the configs in a particular order
-    got_conf = False
     if config.has_section(profile_name):
-        got_conf = True
         result['profile'] = profile_name
         for name, value in config.items(profile_name):
             # note the config_defaults dictionary also serves to indicate which
@@ -2031,11 +2029,6 @@ def read_config(profile_name, user_config=None):
                 else:
                     result[name] = value
 
-    # Check if the specified profile had a config specified
-    if not got_conf:
-        raise ConfigurationError("no configuration for profile name: %s"
-                                 % profile_name)
-
     # special handling for cert defaults
     cert_defaults = {
         'cert': '~/.koji/client.crt',
